plugins/main/public/controllers/management/components/management/configuration/configuration-settings.js

```diff
@@ -80,13 +80,6 @@ export default [
         goto: 'alerts',
         when: 'manager',
       },
-      {
-        name: 'Integrations',
-        description:
-          'Slack, VirusTotal and PagerDuty integrations with external APIs',
-        goto: 'integrations',
-        when: 'manager',
-      },
```

Comment:
**nitpick**: this is related to the configuration of the Wazuh server and this should not be modified.